app/views/shared/article/_file_display.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if cms_block_content(file_field).try(:file)&.present? %>
+  <% file           = cms_block_content(file_field).file %>
+  <% file_language  = cms_block_content(file_language_field).present? ? cms_block_content(file_language_field) : "English" %>
```

Comment:
I think we should probably hide the language label if it hasn't been set rather than using English as default because that could be wrong.